server/viame_tasks/tasks.py

```diff
@@ -255,37 +271,20 @@ def train_pipeline(
                 )
             else:
                 manager.updateStatus(JobStatus.PUSHING_OUTPUT)
-            timestamp = datetime.utcnow().replace(microsecond=0).isoformat()
 
             # This is the name of the folder that is uploaded to the
             # "Training Results" girder folder
-            girder_output_folder_name = f"{pipeline_name} {timestamp}"
-            girder_output_folder = training_output_path / girder_output_folder_name
-
-            # Trained_ prefix is added to conform with existing pipeline names
-            # This is the name that will appear in the client (with trained_ removed)
-            trained_model_folder_name = f"trained_{pipeline_name}"
-            named_training_output = training_output_path / trained_model_folder_name
-
-            # Move the original folder to our new folder
-            shutil.move(str(training_results_path), named_training_output)
-
-            # If `_trained_pipeline_folder()` returns `None`, the results of this
-            # training job will be uploaded to Girder, but will not be runnable as
-            # a normal pipeline through the client
-            trained_pipeline_folder = _trained_pipeline_folder()
-            if trained_pipeline_folder:
-                shutil.copytree(
-                    named_training_output,
-                    trained_pipeline_folder / trained_model_folder_name,
-                )
-
-            # Rename this folder so that it appears properly in girder
-            shutil.move(str(named_training_output), girder_output_folder)
-            gc._uploadFolderRecursive(
-                girder_output_folder, results_folder["_id"], "folder"
+            girder_output_folder = gc.createFolder(
```

Comment:
Training a pipeline with the same name how do we want to handle that?  Right now I believe it errors out because the folder already exists.  I would think it would be nice to be able to keep the same name while testing training and not requiring the user to do 'trainedModel1, trainedModel2, trainedModel3, trainedModel4' or have them delete the previous trained model before running one with the same name.  Open to other options including overwriting, or disabling training with the same name..  In another PR I could check currently trained and at least provide and info/warning to the user that they will be overwriting an existing pipeline with the same name.

Reply:
Yeah, I think warning the user before overwriting is a good approach. I'm hesitant to just silently create another folder that's something like `Trained Model (1)`, as then they might be duplicating work without realizing it, or lose track of what's what.

Reply:
Yeah if you're good with modifying so it will overwrite, we can add another Issue and eventual  PR for warning the user in the UI that they are overwriting an existing pipeline.

Reply:
Sounds good to me. Should we add the server-side overwriting behavior in here now, or wait until that other PR? I'm leaning a bit towards the latter

Reply:
Marking as resolved and added #485